desktop/apps/galleries_institutions/components/primary_carousel/test/template.coffee

```diff
@@ -6,18 +6,36 @@ Profile = require '../../../../../models/profile'
 template = jade.compile fs.readFileSync(filename = require.resolve '../template.jade'), filename: filename
 
 describe 'PrimaryCarousel template', ->
-  beforeEach ->
-    @profile = new Profile fabricate 'profile', owner_type: 'PartnerGallery'
-    partner = @profile.related().owner
-    partner.set fabricate 'partner'
-    partner.related().shows.add fabricate 'show'
 
-  it 'renders correctly', ->
-    @html = template profiles: [@profile]
-    @$ = $.load @html
+  describe 'with at least one profile to show', ->    
+    before ->
+      @profile = new Profile fabricate 'profile', owner_type: 'PartnerGallery'
+      partner = @profile.related().owner
+      partner.set fabricate 'partner'
+      partner.related().shows.add fabricate 'show'
 
-    @$('.gpc-subheadline').text()
-      .should.equal 'Past New York Show'
+    it 'displays the carousel', ->
+      @html = template profiles: [@profile]
+      @$ = $.load @html
+      @$('.gpc-body').should.not.be.empty()
 
-    @$('.gpc-location-dates').text()
-      .should.equal 'New York, Jul 12 – Aug 23, 2013'
+    it 'renders correctly', ->
+      @html = template profiles: [@profile]
+      @$ = $.load @html
+
+      @$('.gpc-subheadline').text()
+        .should.equal 'Past New York Show'
+
+      @$('.gpc-location-dates').text()
+        .should.equal 'New York, Jul 12 – Aug 23, 2013'
+
+    describe 'with more than one profile to show', ->    
+      it 'shows navigation arrows', ->
+        @html = template profiles: [@profile, @profile]
+        @$ = $.load @html
+        @$*('.gpc-bumpers').should.not.be.empty()
+
+  describe 'with no profiles to show', ->
+    it 'does not display the carousel', ->
+      @html = template profiles: []
+      @html.should.be.empty()
```

Comment:
Ahh, such a simple test 😎. 

I also added a few specs above to make the behavior of the template clearer for all the difference cases.